packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added an optional `continueSearch` function to `ISearchOptions` to allow for user prompting and data set search cancellation after the list of data sets to be searched is retrieved. [#2300](https://github.com/zowe/zowe-cli/pull/2300)
```

Comment:
I suggest rewording to something like: Added an optional `continueSearch` function to the `ISearchOptions` prompt/function/whatever. After an initial data set search has been executed, the new function can be set to prompt users to continue the existing search or cancel it.

Reply:
I made a change to try and follow that suggestion.